main_routers/websocket_router.py

```
@@ -0,0 +1,451 @@
+            except Exception as e:
+                logger.warning(f"通知前端失败: {e}")
+        await websocket.close()
+        return
+    
+    this_session_id = uuid.uuid4()
+    async with _lock:
+        global session_id
+        session_id[lanlan_name] = this_session_id
+    logger.info(f"⭐websocketWebSocket accepted: {websocket.client}, new session id: {session_id[lanlan_name]}, lanlan_name: {lanlan_name}")
+    
+    # 立即设置websocket到session manager，以支持主动搭话
```

Ruff (0.14.8)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)